api/db/seeds.rb

```diff
@@ -0,0 +1,71 @@
+puts "Init seed"
+User.destroy_all
+User.create(name: 'Lian Nivin', email: 'liam@kampu.pe', role: "regular", password: '123456')
+User.create(name: 'Cristian Berly', email: 'berli@kampu.pe', role: "owner", password: '123456')
+
+clubs = Club.create([{name: "Club #1", address: 'Jr cayumba 440',
+schedule: {
+  'monday-friday': {
```

Comment:
How do you count `monday, tuesday, wednesday, thrusday and friday` with that?

Reply:
Hey, I assume that the club will be open from Monday to Friday, so I use the key `monday-friday`.